contributing.md

```diff
@@ -0,0 +1,80 @@
+> # Meraki Frontend Repository
+
+This repository contains the source code for the Meraki Frontend. The Meraki Frontend is a web application that allows users to learn python , spoken english and other things.
```

Comment:
I think we might want to lead with a more enticing introduction about who the app will benefit to encourage contributions.

Reply:
Yes i am not a good writer we might need to take help from someone else